applications/fishing-map/src/features/resources/resources.selectors.ts

```diff
@@ -20,7 +26,51 @@ export const selectVisibleResources = createSelector(
   }
 )
 
-export const selectDataviewsResourceQueries = createSelector(
+/**
+ * Prepare dataviews for querying resources, by altering dataset configs
+ * - Filter out non track dvs
+ * - Add thinning query params
+ * - Add extra dataset config for track speed, if needed
+ * - Filter events dataset configs
+ */
+export const selectDataviewsForResourceQuerying = createSelector(
```

Comment:
Would be possible to extract the logic of this selector out to the dataviews-client package so that it can be reused also in other apps consuming dataviews like vessel-viewer?

Reply:
It would be good to discuss this. Right now I'm bothered by the fact that dataviews-client handles what in my view is application logic:
- which type of resources do we want to load from a track dataview?
- what thinning parameters do we want to apply
- what fields do we want to load on the track?
- etc

What can be done though is move some of the barebones dataviews datasetsConfig modification logic as an helper in dataviews-client (in the spirit of https://github.com/GlobalFishingWatch/frontend/pull/803/files#diff-8a29c0005ba549ec031b20d1e6579e73cd0cd5c9e39406770f548b1d9980a442R71)

Reply:
> What can be done though is move some of the barebones dataviews datasetsConfig modification logic as an helper in dataviews-client

Something like `getBaseTrackDataviewDatasetConfigs` if yo know what I mean

Reply:
Makes sense moving them to a helper in dataviews-client that could accept those per app configurations as arguments (type of resources, thinning parameters, etc..)